document/RuleMetadataProfile.mdk

```diff
@@ -1 +1,165 @@
+the stable `id` property unchanged).
+
+~ IsoExample
+`"SpecifyMarshalingForPInvokeStringArguments"`
+~
+
+### `abstract` property { #ruleDescriptor-abstract }
+
+A `ruleDescriptor` object may contain a property named `abstract` whose value is a string containing
+a concise description of the rule. The `abstract` property should be a single sentence that is understandable
+when displayed in user interface contexts where the available space is limited to a single line of text.
+
+~ IsoExample
+`"Specify marshaling for P/Invoke string arguments"`
+~
+
+### `description` property { #ruleDescriptor-description}
+
+An `ruleDescriptor` object should contain a property named `description` whose value is a string that describes the rule.
+
+The `description` property should, as far as possible, provide details sufficient to enable resolution of issues. 
+
+The first sentence of the `description` property should provide a concise description of the rule,
+suitable for display in cases where UI is limited.
+Tools that construct `description` in this way need not provide a value for the `abstract` property.
+Tools that do not construct `abstract` in this way should provide a value for the `abstract` property,
+because otherwise, the initial portion of `description` that a viewer displays where UI space is limited
+might not be understandable.
+
+### `options` property { #ruleDescriptor-options }
+
+A `ruleDescriptor` object may contain a property named `options`, whose value is a JSON object
+consisting of an arbitrary set of string/value pairs.
+When present, the `options` objects shall describe the set of configurable options supported by the rule.
+
+The value within each string-value pair shall be a string, which may be the empty string.
+The value shall not be a JSON array or sub-object.
+
+### `formatSpecifiers` property { #ruleDescriptor-formatSpecifiers }
+
+A `ruleDescriptor` object may contain a property named `formatSpecifiers`, whose value is a JSON object
+consisting of an arbitrary set of string/value pairs.
+When present, the `formatSpecifiers` object shall describe a set of message formatting strings
+that have or may be returned by the tool.
+The string name of each format string shall be an identifier that uniquely identifies each member.
+The value within each string-value pair shall be a string that can be passed as a control parameter to
+string formatting API (_e.g._, `printf`) in order to construct a formatted message in combination with
+an arbitrary number of additional data parameters.
+
+The `formatSpecifiers` property contents may be limited to values that are associated with issues
+written to the issue log.
+In this use, the relevant issue objects must provide a `formattedMessage` (&sect;[#formattedMessage]) object
+that refers to an associated messageFormatString. In another use, the `formatSpecifiers` property 
+contents may be used to document all possible message format strings that can be returned by a rule.
```

Comment:
your position is credible. i'd thought that we could think of the rules metadata publishing scenario as a (greatly stripped down) profile but, since SA results are entirely omitted perhaps you're right that this isn't the place.